### PR TITLE
add --ip= to docker run flags, for #264

### PR DESCRIPTION
**Pull request by dreamcat4**
This PR is for missing docker run flag `--ip=`

https://github.com/michaelsauter/crane/issues/264

### CODE REVIEW

**Comment by dreamcat4** [comment]
Uh. pls forgive the whitespace formatting.

File: README.md [hunk 1]
```diff
@@ -119,6 +119,7 @@ The map of containers consists of the name of the container mapped to the contai
 	* `group-add` (array) Need Docker >= 1.8
 	* `hostname` (string)
 	* `interactive` (boolean)
+	* `ip` (string) ip address
```
Comment (bjaglin) [inline README.md]: if you had `ip`, you might as well add its IPv6 counterpart

Comment (bjaglin) [inline README.md]: Also, please specify the minimal Docker version `Need Docker >= 1.x`

**Comment by bjaglin** [comment]
also, can you squash your commits? thanks!

**Comment by michaelsauter** [comment]
Thanks for this!

**Comment by dreamcat4** [comment]
OK will do.

**Comment by dreamcat4** [comment]
@bjaglin OK its squashed now. Sorry for the waiting. Done.

**Comment by bjaglin** [comment]
Lovely!

**Pull request merged**